samples/DataSourceWithProcessingAndExtensions/Sample/TestingAssemblyInfo.cs

```diff
@@ -0,0 +1,5 @@
+// © Microsoft Corporation. All rights reserved.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("DataGenerator")]
```

Comment:
This seems weird to have in a sample. Why not just make whatever internal method/property needed by DataGenerator public?